### PR TITLE
Add 2.19 to language evolution page

### PR DESCRIPTION
**Pull request by MaryaBelanger**
Adds a 2.19 entry to the [Language evolution](https://dart.dev/guides/language/evolution) page.

**Staged:** https://dart-dev--pr4544-2-19-lang-evolution-qw79orzf.web.app/guides/language/evolution#dart-219

### CODE REVIEW

**Review by parlough — APPROVED** [review]
Thanks for working on this, looks fantastic to me after the adjustments from the old PR :D

File: src/_guides/language/evolution.md [hunk 1]
```diff
@@ -284,6 +284,32 @@ see [Adding features to a class: mixins][] in the language tour.
 
 [Adding features to a class: mixins]: /guides/language/language-tour#adding-features-to-a-class-mixins
 
+### Dart 2.19
+
+Dart 2.19 introduced some precautions surrounding type inference.
```
[the remark this answers is not in the document]
Comment (MaryaBelanger) [reply src/_guides/language/evolution.md:289]: I got the info from the changelog, this is the very light summary of that 😅 I'll simplify it though

[the remark this answers is not in the document]
Comment (MaryaBelanger) [reply src/_guides/language/evolution.md:289]: @mit-mit better?

[the remark this answers is not in the document]
Comment (mit-mit) [reply src/_guides/language/evolution.md:289]: This is great, thanks

**Pull request merged**